/nas/student/NicolasKolbenschlag/emotion_uncertainty_bachelorarbeit/MuSe-LSTM-Attention-baseline-model/emotion_recognition/main_uncertainty_ensemble.py --feature_set bert-4 --emo_dim_set valence --epochs 1 --refresh --n_seeds 2 --seed 314 --attn --rnn_bi --loss ccc --uncertainty_approach ensemble_averaging --attn_dr .5 --out_dr .5 --rnn_n_layers 2 --rnn_dr .5 --measure_uncertainty_globally --global_uncertainty_window 20 --normalize_uncalibrated_global_uncertainty_measurement --load_subjectivity_from_file
Constructing dataset and data loader ...
Constructing data from scratch ...
Subjectivities deserialized from file.
Samples in partitions: (3132, 62, 64)
Input feature dim: 768.
==================================================
Training model... [seed 314]
Seed 314 | Best [Val CCC]: 0.0243 [' 0.0243']| Loss: 0.9756 | PCC: 0.1523 ['0.1523'] | RMSE: 0.7830 ['0.7830']
On Test: CCC  0.0552 | PCC  0.2744 | RMSE  0.7507
==================================================
Training model... [seed 315]
Seed 315 | Best [Val CCC]: 0.0966 [' 0.0966']| Loss: 0.9035 | PCC: 0.1950 ['0.1950'] | RMSE: 0.3633 ['0.3633']
On Test: CCC  0.1048 | PCC  0.2644 | RMSE  0.3998
==================================================
--------------------TEST--------------------
Confirmed prediction ccc-score: 0.18395565751420467
Traceback (most recent call last):
  File "/nas/student/NicolasKolbenschlag/emotion_uncertainty_bachelorarbeit/MuSe-LSTM-Attention-baseline-model/emotion_recognition/main_uncertainty_ensemble.py", line 327, in <module>
    main(params)
  File "/nas/student/NicolasKolbenschlag/emotion_uncertainty_bachelorarbeit/MuSe-LSTM-Attention-baseline-model/emotion_recognition/main_uncertainty_ensemble.py", line 250, in main
    uncertainty_utilities_global.evaluate_uncertainty_measurement_global(params, ENSEMBLE, data_loader["test"], data_loader["devel"])
  File "/nas/student/NicolasKolbenschlag/emotion_uncertainty_bachelorarbeit/MuSe-LSTM-Attention-baseline-model/emotion_recognition/uncertainty_utilities_global.py", line 333, in evaluate_uncertainty_measurement_global
    def evaluate_uncertainty_measurement_global(params, model, test_loader, val_loader):
  File "/nas/student/NicolasKolbenschlag/emotion_uncertainty_bachelorarbeit/MuSe-LSTM-Attention-baseline-model/emotion_recognition/uncertainty_utilities_global.py", line 266, in evaluate_uncertainty_measurement_global_help
    print(full_subjectivities_pred)
AssertionError: (4318, 1) != (4218, 1)
